pkg/local_object_storage/pilorama/boltdb.go

```diff
@@ -625,8 +625,20 @@ func (t *boltForest) TreeGetOpLog(cid cidSDK.ID, treeID string, height uint64) (
 }
 
 // TreeDrop implements the pilorama.Forest interface.
+// In case of empty treeID drops all trees related to container.
```

Comment:
Lets maybe move it to `pilorama.Forest` interface?

Reply:
Agree. Moved.